lib/screens/dashboard.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:bytebank/components/features/lista.dart';
+import 'package:bytebank/components/quadro.dart';
+import 'package:bytebank/screens/lista_transferencias.dart';
+import 'package:flutter/material.dart';
+
+import 'lista_contatos.dart';
+
+const _imagemDashboard = 'assets/logo-bytebank.png';
+
+const _tituloAppBar = 'Dashboard';
+
+class Dashboard extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(_tituloAppBar),
+      ),
+      body: Container(
+        color: Colors.white,
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: <Widget>[
+            Quadro(
+              altura: 400.0,
+              caminhoImagem: _imagemDashboard,
+            ),
+            ListaFeatures(
+              quandoClicaContatos: () => _vaiParaListaContatos(context),
+              quandoClicaTransacoes: () => _vaiParaListaTransferencias(context),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  //TODO todas as navegações eu usei esse estilo simples
+  _vaiParaListaContatos(BuildContext context) {
```

Comment:
This is also about the responsibility/layering I mentioned about
This UI widget knows how to navigate to other screens, isn't there any kind of Navigator that is able to coordinate these transitions?

As a rule, we always try to make UI views as simple as possible, only showing whatever it is given to them.

Reply:
In this project, I dont will focus on techniques about navigation good practicies, I writed the simplest code to do navigate between screens. If there are another approach simplest than it, can you show me a example?

Reply:
Do you is referring about the route configuration, right? I see in [documentation](https://flutter.dev/docs/cookbook/navigation/named-routes) the possibilities to configure the routes and I'm plainning to speak about this topic together the navigation layer in next module, ok?

Reply:
Exactly